<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Install MT Model</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,1,1">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>MT Model Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Model Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="editName">
        <property name="text">
         <string>SampleModel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="title">
      <string>Install From ...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="grpBoxInternet">
        <property name="title">
         <string>From Internet ...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="editInternet">
           <property name="text">
            <string>http://www.statmt.org/~jie/models/EnFr4MosesGUI.zip</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpBoxLocal">
        <property name="title">
         <string>From Local File</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="9,1">
         <item>
          <widget class="QLineEdit" name="editLocal">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLocal">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="title">
      <string>Install To ...</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="9,1">
      <item>
       <widget class="QLineEdit" name="editPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>grpBoxInternet</tabstop>
  <tabstop>editInternet</tabstop>
  <tabstop>grpBoxLocal</tabstop>
  <tabstop>editLocal</tabstop>
  <tabstop>btnLocal</tabstop>
  <tabstop>editPath</tabstop>
  <tabstop>btnPath</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
